clerk/organizations_test.go

```diff
@@ -286,6 +286,27 @@ func TestOrganizationsService_UpdateLogo(t *testing.T) {
 	}
 }
 
+func TestOrganizationsService_DeleteLogo(t *testing.T) {
+	client, mux, _, teardown := setup("token")
+	defer teardown()
+
+	organizationID := "org_123"
+	mux.HandleFunc(
+		fmt.Sprintf("/organizations/%s/logo", organizationID),
+		func(w http.ResponseWriter, req *http.Request) {
+			testHttpMethod(t, req, http.MethodDelete)
+			testHeader(t, req, "Authorization", "Bearer token")
+			fmt.Fprint(w, `{"id":"img_1234"}`)
```

Comment:
❓ Doesn't the response contain the organization? That's only for semantics though, but if that's the case, just to avoid confusion, consider `img_1234` -> `org_123` as id?

Reply:
You're right, thanks for catching this. Updated.

Reply:
🚀